nearcore/src/config.rs

```diff
@@ -1016,15 +1016,32 @@ pub fn init_configs(
     Ok(())
 }
 
-pub fn create_testnet_configs_from_seeds(
+/// Params specific to a localnet node, used for configuring the node for certain roles.
+struct LocalnetNodeParams {
+    /// If true, this node is used as a boot node.
+    is_boot: bool,
+    /// If true, this is a validator node.
+    is_validator: bool,
+    // If true, this is an archival node.
+    is_archival: bool,
+    // If true, this is an RPC node.
+    is_rpc: bool,
+}
+
+pub fn create_localnet_configs_from_seeds(
```

Comment:
nit: Can you add a comment explaining the arguments please?

Reply:
done